config/routes.rb

```diff
@@ -69,7 +69,7 @@
     resources :contacts, only: %i[edit update]
 
     # TODO: Extract year constraint to future proof for future cycles
-    resources :recruitment_cycles, param: :year, constraints: { year: /2021|2022/ }, path: "", only: :show do
+    resources :recruitment_cycles, param: :year, constraints: { year: /#{Settings.current_cycle}|#{Settings.current_cycle + 1}/ }, path: "", only: :show do
```

Comment:
can we have a tests like 

https://github.com/DFE-Digital/teacher-training-api/blob/0dff3249fb4dd2150b1a4863f63ddb3ca08b96ea/spec/config/settings_spec.rb#L11

please